inst/odin/carehomes.R

```diff
@@ -482,6 +483,10 @@ initial(cum_admit_by_age[]) <- 0
 
 ## User defined parameters - default in parentheses:
 
+## Parameters of the S classes
+rel_susc[] <- user()
+dim(rel_susc) <- N_vacc_classes
```

Comment:
Ah I see why you had the above. The way to do this nicely with odin (all 3 lines) is:

```r
rel_susc[] <- user()
dim(rel_susc) <- user() # this says - take the length as whatever was provided by the user
N_vacc_classes <- length(rel_susc)
```